### PR TITLE
updated planet mojo project tokens

### PR DESCRIPTION
**Pull request by moskalyk**

### CODE REVIEW

File: index/mainnet/erc721.json [hunk 1]
```diff
@@ -33589,7 +33589,6 @@
       "name": "mojo_war_banners",
       "standard": "erc721",
       "symbol": "pmwb",
-      "logoURI": "https://i.seadn.io/s/raw/files/b05371e10c5ad8efdeab1a9e00b942bf.png?auto=format&dpr=1&w=2048",
```
Comment (PhABC) [inline index/mainnet/erc721.json:33592]: Why was this removed?

Comment (moskalyk) [reply index/mainnet/erc721.json:33592]: I think I added it as a mistake, it's a duplicate image and I don't think it makes sense given the media nature to add it as a `logoURI`

**Review by PhABC — APPROVED** [review]
